### DISCUSSION TITLE
Scoring in GLiNER Paper: threshold?

### DISCUSSION THREAD

**Discussion by davidress-ILW** [urchade/GLiNER#100] [Q&A] [answered]
Hello!

GLiNER is really incredible! Congratulations!

I have read through the paper (GLiNER: Generalist Model for Named Entity Recognition using Bidirectional Transformer), but I did not see what threshold value was used in producing the Precision, Recall and F-1 scores documented in sections 4 and 5?

I ask as the GLiNER implementation defaults the threshold to 0.5 in `model.predict_entities()`, and I was wondering which threshold was used in the paper scoring, as I interpret the paper scores as the baseline capability of GLiNER. Fir example, if I go look at predicting entities of car/vehicle types using a threshold of 0.8, then produce scoring results, I would also like to do the same with the baseline threshold from the paper.

Thank you,
David

**Comment by urchade** [comment, marked as answer]
I have used the default `0.5` threshold value

**Reply by davidress-ILW** [reply]
Thank you. This software is great.

**Reply by urchade** [reply]
Thank you 😊😊
